book/website/ethical-research/ethics-open-source-governance.md

```diff
@@ -0,0 +1,99 @@
+(er-ethics-open-source-governance)=
+# Ethical Considerations when Choosing an Open Source Governance Model
+
+(er-ethics-open-source-governance-summary)=
+## Summary
+
+Open source projects are rooted in ethical principles. 
+Sharing the code so that anyone can use it, welcoming contributors and creating communities around specific projects are the core of open source. 
+But open source communities also have the responsibility to ensure they are a safe, inclusive environment, understand who might be affected by their projects or check that their tools are not being misused. 
+Therefore, Open Source Governance Models should embed those ethical considerations within their structure, but the framework to do so is far from well-established. 
+This chapter discusses some of the key points to take into consideration when designing and implementing ethical guidelines in governance models.
+
+
+(er-ethics-open-source-governance-motivation)=
+## Motivation and Background
+
+When setting up open source projects, it is important to pick a structure for organising the community that takes into account ethical considerations 
```

Comment:
```suggestion
When setting up open source projects, it is important to pick a structure for organising the community that takes into account ethical considerations.
```

Reply:
This sentence feels a bit vague. What do we mean by structure here (leadership, organisation legal status etc.)? How do we know what structure takes ethical considerations into account?